yarn.lock

```diff
@@ -3,23 +3,23 @@
 
```

Comment:
Big yarn.lock change here, but I installed the extension locally and ran all the Cursorless tests so I think we're ok.  Hoping all these updates make dependabot happy 🤞